examples/convert_group_studies.py

```diff
@@ -71,8 +71,15 @@
         bids_basename = make_bids_basename(subject=str(subject_id),
                                            session='01', task='VisualFaces',
                                            run=str(run))
+        '''
+        By using the same anonymization `daysback` number we can
+        preserve the longitudinal structure of mulitple sessions for a
+        single subject and the relation between subjects. Be sure to
+        change or delete this number before putting code online, you
+        wouldn't want to inadvertently de-anonymize your data.
+        '''
```

Comment:
I think it's not kosher to use a multiline comment. See: https://www.python.org/dev/peps/pep-0008/#block-comments